raiden_libs/test/mocks/client.py

```diff
@@ -2,15 +2,19 @@
 from typing import Dict
 from functools import wraps
 
+import gevent
 from eth_utils import is_checksum_address, is_same_address, encode_hex, decode_hex
+
 from web3.utils.events import get_event_data
 from web3 import Web3
 from web3.contract import Contract
 
 from raiden_contracts.contract_manager import get_event_from_abi
 
+from raiden_libs.transport import Transport
 from raiden_libs.utils import private_key_to_address, make_filter, sign_data
-from raiden_libs.messages import BalanceProof, MonitorRequest, FeeInfo, PathsRequest
+from raiden_libs.messages import BalanceProof, MonitorRequest, FeeInfo, PathsRequest, Message, \
```

Comment:
follow the style guide

Reply:
Coding style: https://github.com/raiden-network/raiden/blob/master/CONTRIBUTING.md#coding-style